codebase/config/sync/field.storage.node.field_unique_id.yml

```diff
@@ -0,0 +1,25 @@
+uuid: baf4e3af-f797-4eef-bdf9-32926577fa99
+langcode: en
+status: true
+dependencies:
+  module:
+    - field_permissions
+    - node
+third_party_settings:
+  field_permissions:
+    permission_type: public
+id: node.field_unique_id
+field_name: field_unique_id
+entity_type: node
+type: string
+settings:
+  max_length: 255
+  is_ascii: false
+  case_sensitive: false
```

Comment:
set `case_sensitive` to true, for the sake of avoiding confusion, especially if non-ascii characters are allowed?